app/components/embed/file_row_component.html.erb

```diff
@@ -16,8 +16,10 @@
   <td role="gridcell"><%= file_size_text%></td>
   <td role="gridcell">    
     <% if download? %>
-      <a href="<%= file_url %>" target='_blank' rel='noopener noreferrer' download=''>
-        <i class='sul-i-download-3'></i> Download
+      <a href="<%= file_url %>" target='_blank' rel='noopener noreferrer' download>
+        <i class='sul-i-download-3'></i>
+        <span>Download</span>
+        <span class="sul-embed-sr-only"><%= title %></span>
```

Comment:
without this, the accessible text for all the links was just "download", but they go to different places.